18_child_ITN.do

```diff
@@ -7,6 +7,67 @@
 * 1) check the maximum value for tnln and use it as a maximum value for your "forvalues" of `t' and `j'
 * 2) check the tn15_* to find out the maximum amount of person recorded under each bednet and use it to determine the number of time you are going to replicate your "forvalues"
 
+	// CentralAfricanRepublic2018: tn5 - 11/12/13/14/15/16:other written down/36: other
+	// tnln - 1~10
+	// max value of tnln - 10; tn15_1-20; tn15_2-23; tn15_3-22; tn15_4-22
+if inlist(country_name,"CentralAfricanRepublic2018") {
```

Comment:
Section has not been updated from Guinea-Bissau2018 version, but top-banner description of key variables have been for CAR2018. The logic here is still very much murky, now that tn15_1-4 are slightly more inconsistent than before. 
@wengxyu1030

Reply:
@wengxyu1030 Just realised this one here...
ITN treatment for CAR2018 was quite elusive.. If you have a chance to look at it, may I get some thoughts from you about it?